package/Makefile

```diff
@@ -45,7 +45,7 @@ CLEAN_FLIGHT := YES
 endif
 
 # Set up targets
-ALL_BOARDS        := coptercontrol pipxtreme revomini sparky sparky2 sparkybgc quanton colibri flyingf3 flyingf4 aq32
+ALL_BOARDS        := coptercontrol pipxtreme revomini sparky sparky2 sparkybgc quanton colibri flyingf3 flyingf4 aq32 naze32
```

Comment:
Yah.. But I needed it anyway on the branch to get windows packaged builds.

Just as an aside packaging is broken everywhere-- on OSX by the QT5.5 change, by this.. That's something I'm going to look at systemically in a week or two.